runtime/jilgen/jilconsts.c

```diff
@@ -321,9 +321,11 @@ writeConstants(OMRPortLibrary *OMRPORTLIB, IDATA fd)
 			writeConstant(OMRPORTLIB, fd, "J9TR_ELS_machineSPSaveSlot", offsetof(J9VMEntryLocalStorage, machineSPSaveSlot)) |
 #endif /* J9VM_PORT_ZOS_CEEHDLRSUPPORT */
 
-#if defined(OMR_GC_CONCURRENT_SCAVENGER)
+#if defined(J9VM_ARCH_AARCH64) || defined(J9VM_ARCH_ARM) || defined(J9VM_ARCH_POWER)
```

Comment:
Why don't all platforms include j9cfg.h?

Reply:
Should the build warning be an error instead?

Reply:
That's a good idea. I'll see if we can do that.

Reply:
I was going to make that change, but it appears it will be predicated on addressing numerous warning in compiler code, which is a much bigger undertaking. Please merge this now if you're satisfied.

Reply:
This does seem like a change we should get into the 0.33 release, ideally before the weekend.

Reply:
I'd be happy to help get this in 0.33, but it only deals with a warning: I don't think there are any other effects on the build and no consequences for users, so it wouldn't strictly be necessary.

Reply:
ok, I misunderstood the problem then. I thought the constant would have the wrong value.

Reply:
If it doesn't matter, why do we even bother with a warning?

Reply:
We don't issue the warning, the compiler does.
I think the code only cares whether the macro is defined or not.